camelot/core/naming.py

```diff
@@ -768,4 +828,35 @@ def new_context(self) -> NamingContext:
         """
         return NamingContext()
 
+    def _bind_object(self, obj):
+        """
+        Helper method for binding any type of python object under the appropriate name.
+        This functionality is meant for backend binding of objects and will always perform a mutable bind.
+
+        :param obj: the object to be bound.
+
+        :return: the full qualified composite name of the bound object, relative to the initial naming context.
+
+        :raises:
+            UnboundException NamingException.unbound: if this NamingContext has not been bound to a name yet.
+        """
+        from camelot.core.orm import Entity
+        if obj is None:
+            return ('constant', 'null')
+        if isinstance(obj, bool):
+            return ('constant', 'true' if obj else 'false')
+        if isinstance(obj, (str, int, Decimal)):
+            return ('constant', type(obj).__name__.lower(), str(obj))
+        if isinstance(obj, Entity):
+            # TBD: possibly move the context specific object validations to the respective context?
+            if not inspect(obj).persistent or obj.id is None:
```

Comment:
cfr de entity admin, is er logica om de composite primary key te 'extraheren'

Reply:
https://github.com/v-finance/camelot/pull/336/